contexts-config.json

```diff
@@ -0,0 +1,32 @@
+{
+    "currentContextUuid": {
+      "value": "{UUID}"
+    },
+    "contexts": [
+      {
+        "uuid": {
+          "value": ""
+        },
+        "name": "default",
+        "localOnlyContextV0": {}
+      },
+      {
+        "uuid": {
+          "value": "{UUID}"
+        },
+        "name": "{NAME}",
+        "remoteContextV0": {
+          "host": "{IP}",
+          "remotePortalPort": 9720,
+          "kurtosisBackendPort": 9722,
+          "tunnelPort": 9721,
+          "tlsConfig": {
+            "certificateAuthority": "{CA}",
+            "clientCertificate": "{CLIENT_CERT}",
+            "clientKey": "{CLIENT_KEY}"
+          },
+          "envVars": "{}"
```

Comment:
The AWS env variables need to be added.

Reply:
@laurentluce This is just for my curiosity, but what's the need to supply a context config? Is it not enough to intall kurtosis-cli and then run `kurtosis engine start`?  Ie. does this instance actually need the remote context config on the machine?

Reply:
Answered in other communicaton